code/game/turfs/simulated/water.dm

```diff
@@ -15,3 +15,20 @@
 	barefootstep = FOOTSTEP_WATER
 	clawfootstep = FOOTSTEP_WATER
 	heavyfootstep = FOOTSTEP_WATER
+
+/turf/open/water/decorative
```

Comment:
you should remove all of this except the initial_gas_mix and planetary_atmos lines

Reply:
Okay, done I think?